UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentFileShareService.cs

```diff
@@ -137,5 +137,23 @@ public async Task<bool> CommitLargeMediaExchangeSet(string batchId, string excha
         {
             return await fileShareService.CommitAndGetBatchStatusForLargeMediaExchangeSet(batchId, exchangeSetZipPath, correlationId);
         }
+
+        public async Task<List<BatchFile>> SearchInfoFilePath(string batchId, string correlationId)
+        {
+            var uri = $"{fileShareServiceConfig.Value.BaseUrl}/batch?$filter=$batch{fileShareServiceConfig.Value.ProductType} BusinessUnit eq '{fileShareServiceConfig.Value.BusinessUnit}' and $batch(Content) eq '{fileShareServiceConfig.Value.ContentInfo}'";
+            return await fileShareService.SearchFolderDetails(batchId, correlationId, uri);
+        }
+
+        public async Task<List<BatchFile>> SearchAdcFilePath(string batchId, string correlationId)
+        {
+            var uri = $"{fileShareServiceConfig.Value.BaseUrl}/batch?$filter={fileShareServiceConfig.Value.ProductType} businessUnit eq '{fileShareServiceConfig.Value.BusinessUnit}' and $batch(Content) eq '{fileShareServiceConfig.Value.Content}' and $batch(Catalogue Type) eq '{fileShareServiceConfig.Value.Adc}'";
+            return await fileShareService.SearchFolderDetails(batchId, correlationId, uri);
```

Comment:
there seems to be similarities with these two methods

Reply:
Resolved